### PR TITLE
M4: add slides for methodology

### PR DESCRIPTION
**Pull request by RazvanN7**

### CODE REVIEW

File: website/docs/events/m4-slides/.slides/methodology.md [hunk 1]
```diff
@@ -1,25 +1,81 @@
...
+`-- chapterN
+```
 
 ----
 
-### TODO
+### Technologies used
 
-TODO
+- Github for storing content
```
Comment (razvand) [inline website/docs/events/m4-slides/.slides/methodology.md]: I would replace them with logos.

File: website/docs/events/m4-slides/.slides/methodology.md [hunk 2]
```diff
@@ -1,25 +1,81 @@
...
+- guidelines on content organization
+- guidelines on contribution procedures
+- technologies used
 
-probably follow structure in [the `methodology` repository](https://github.com/open-education-hub/methodology/), apart from infrastructure
+----
+
+### Types of Content
+
+- reading
```
Comment (razvand) [inline website/docs/events/m4-slides/.slides/methodology.md:27]: Replace them with pics, logos, such as those here: https://icons8.com/

File: website/docs/events/m4-slides/.slides/methodology.md [hunk 3]
```diff
@@ -1,25 +1,81 @@
...
 ## Methodology
 
-Owner: RăzvanN
+----
+
+### Methodology
+
+- content creation
+- infrastructure used for deployment, testing and publishing
+
```
Comment (razvand) [inline website/docs/events/m4-slides/.slides/methodology.md:11]: I would add a slide named "Scope" or "Questions Answered". And add to the slides the questions that the methodology provides answers too. This is very useful to hook the audience, as these are likely questions they are asking themselves.

File: website/docs/events/m4-slides/.slides/methodology.md [hunk 4]
```diff
@@ -1,25 +1,81 @@
...
+----
+
```
Comment (razvand) [inline website/docs/events/m4-slides/.slides/methodology.md:13]: Before you venture into internals of content, storing, types, I would go two higher level use cases.

1. **Use content**: I am an educator, I am assigned to deliver a class, I want to have a repository set with all I need. Methodology: OK, here's what you do: fork repository, configure, add logistics info, deploy, use automation, mentor student, ask student to contribute.
1. **Develop content**: I am an educator, I want to develop or port a new class. I want to create content that I, and others will use. Probably no "I" but "we" - multiple educators develop said content. Methodology: OK, here's what you do: fork oer-template, start developing content according to content types, look at existing content, build along the way, use linters, verify, ask for compliance with OpenEdu guidelines, have it hosted in OpenEdu, with credits given to you. Profit.

File: website/docs/events/m4-slides/.slides/methodology.md [hunk 5]
```diff
@@ -1,25 +1,81 @@
...
+### Content Creation
+
+- types of content
```
Comment (razvand) [inline website/docs/events/m4-slides/.slides/methodology.md:16]: Add a diagram in draw.io that lists all of these items interconnected.

File: website/docs/events/m4-slides/.slides/methodology.md [hunk 6]
```diff
@@ -1,25 +1,81 @@
...
+### Technologies used
 
-TODO
+- Github for storing content
+- Markdown for everything
+- Draw.io for images and diagrams
+- reveal-md for slide rendering
+- others
 
 ----
```
Comment (razvand) [inline website/docs/events/m4-slides/.slides/methodology.md:77]: Add a section called "Challenges and Road Ahead" or something along that way. Mention things we are unsure of, and items we want / need to tackle in the future for the methodology.

**Review by razvand — CHANGES_REQUESTED** [review]

File: website/docs/events/m4-slides/.slides/methodology.md (hunk 3 above)
Comment (RazvanN7) [reply website/docs/events/m4-slides/.slides/methodology.md:11]: I have a narrative where this is explained, but it's not explicit on the slides.

**Review by razvand — APPROVED** [review]

**Pull request merged**